src/views/verify.tsx

```diff
@@ -135,9 +159,17 @@ export const Verify: React.FunctionComponent = () => {
           try {
             if (rawDocument) {
               setStatus("PENDING");
+              const supportedNetwork = supportedNetworks.find((item) => item.network === network);
+
+              if (supportedNetwork === undefined) {
+                throw new Error("Supported network not found!");
+              }
+
+              const provider = supportedNetwork.provider;
               const verify = verificationBuilder(openAttestationVerifiers, {
-                provider: new providers.InfuraProvider(network),
+                provider: provider(),
```

Comment:
Is the `provider` variable used anywhere else? Maybe we can just directly use `supportedNetwork.provider()`?